src/main/java/baseball/receiver/ConsoleInputReceiver.java

```diff
@@ -0,0 +1,61 @@
+package baseball.receiver;
+
+import baseball.number.GameNumber;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class ConsoleInputReceiver implements InputReceiver {
+
+    private final Scanner scanner;
+    private final int START_GAME = 1;
+
```

Comment:
```suggestion
public class ConsoleInputReceiver implements InputReceiver {
    private static final int START_GAME = 1;

    private final Scanner scanner;
```

상수는 클래스 최상위에 private static final을 사용합니다 🙂